index.html

```diff
@@ -34,163 +34,177 @@
       crossorigin="anonymous"
       referrerpolicy="no-referrer"
     />
-
-    <link rel="stylesheet" href="./styles/style.css" />
-    <link rel="stylesheet" href="./styles/home.css" />
+    <link rel="stylesheet" href="style.css" />
     <title>Global Remote Developer Summit</title>
   </head>
   <body>
-    <header class="desktop">
-      <div class="sub-header">
-        <a href="#"><i class="fa-brands fa-facebook"></i></a>
-        <a href="#">
-          <i class="fa-brands fa-twitter"></i>
-        </a>
-
-        <a href="#">English</a>
-        <a href="#">My page</a>
-        <a href="#">Log out</a>
-      </div>
+    <header>
       <nav>
-        <div class="logo">
-          <h3>WeRemote</h3>
+        <div class="top-nav">
+          <div class="top-nav__container">
+            <img
+              src="./images/home/facebook.svg"
+              alt="facebook-logo"
+              class="icon"
+            />
+            <img
+              src="./images/home/twitter.svg"
+              alt="facebook-logo"
+              class="icon"
+            />
+            <ul class="top-nav__menu">
+              <li class="top-nav__menu-item"><a href="#">English</a></li>
+              <li class="top-nav__menu-item"><a href="#">My Page</a></li>
+              <li class="top-nav__menu-item"><a href="#">Logout</a></li>
+            </ul>
+          </div>
         </div>
-        <ul class="parent-link">
-          <li><a href="./index.html">Home</a></li>
-          <li><a href="./about.html">About</a></li>
-          <li>Programe</li>
-          <li>Join</li>
-          <li>Sponser</li>
-          <li>News</li>
-          <li><a class="highlight" href="">WeRemote</a></li>
-        </ul>
-      </nav>
```

Comment:
- [ ] As mentioned by the previous reviewer, your `top-nav` should be aligned with the page menu to follow the provided design. Therefore kindly add some `margin-right` to your `top-nav__container` class to match Behance Design.

### Your design :point_down: 
![Screenshot from 2023-04-22 15-55-43](https://user-images.githubusercontent.com/91142432/233794871-773d3cef-71a5-470a-96ff-f658a1027f62.png)

### Provided Design :point_down: 
![Screenshot from 2023-04-22 16-07-05](https://user-images.githubusercontent.com/91142432/233794953-9d2b6f29-b1b9-4a6b-851f-abf78c7b904d.png)